src/libraries/Microsoft.Extensions.Hosting.Systemd/src/SystemdHelpers.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.Extensions.Hosting.Systemd
     /// </summary>
     public static partial class SystemdHelpers
     {
+        internal const string NOTIFY_SOCKET_ENVVAR_KEY = "NOTIFY_SOCKET";
```

Comment:
nit: drop the `_ENVVAR_KEY` suffix, and them it `private`.

Reply:
I made them internal so that the NOTIFY_SOCKET magic string could be shared with SystemdNotifier. Happy to revert that part and make the strings private if you'd prefer to just have the literal defined twice.

Reply:
Actioned